src/resources/authoring/gv-1.json

```diff
@@ -2965,12 +2965,22 @@
       "comment" : "*** Level 3, Mission 3.1: Charcoal -> Frost (Albino), Size - Working Everything ***",
       "container" : "FVContainer",
       "template" : "ZoomChallenge",
+      "scoreMetrics": [
+        "disc_clicked",
+        "disc_destroyed",
+        "protein_broken"
+        ],
```

Comment:
As mentioned above, if `scoreMetrics` (or `scoreWeights`) were an object of the form:
```javascript
"scoreWeights": {
  "disc_clicked": 1,
  "disc_destroyed": 1,
  "protein_broken": 1
},
```
then the scoring would be a bit more flexible. In either case, this should ultimately be added to the schema for the zoom challenges (once that schema is written).